metricflow/model/semantics/semantic_model_lookup.py

```diff
@@ -41,27 +41,31 @@
 
 
 class SemanticModelLookup(SemanticModelAccessor):
-    """Tracks semantic information for semantic model held in a set of SemanticModelContainers.
+    """Tracks semantic information for semantic models held in a set of SemanticModelContainers.
 
-    This implements both the SemanticModelAccessors protocol, the interface type we use throughout the codebase.
+    This implements the SemanticModelAccessors protocol, the interface type we use throughout the codebase.
     That interface prevents unwanted calls to methods for adding semantic models to the container.
     """
 
-    def __init__(  # noqa: D107
+    def __init__(
         self,
         model: SemanticManifest,
     ) -> None:
-        self._model = model
+        """Initializer.
```

Comment:
Does this docstring do anything except to tell you that "model" actually means "semantic manifest"? (Probably a holdover from Transform.)
In which case should we just rename the param and delete the docstring?

Reply:
Not my finest work, I admit. 🤣  

I think I was trying to say that this is how we load all of the models into the lookup class, as in any test cases or whatever that want to adjust the model set should just initialize something with a custom semantic manifest. 

Also I totally meant to change that to semantic_manifest but I clearly forgot. I'll fix this in a follow-up, there's another tweak I wanted to make to MetricLookup around this as well, and I may take a shot at getting rid of the protocols as well.